scripts/deployment/phase2-assets/collaterals/deploy_cbeth_collateral.ts

```diff
@@ -64,7 +65,7 @@ async function main() {
         maxTradeVolume: fp('1e6').toString(), // $1m,
         oracleTimeout: '3600', // 1 hr,
         targetName: hre.ethers.utils.formatBytes32String('ETH'),
-        defaultThreshold: fp('0.02').add(oracleError).toString(), // ~4.5%
+        defaultThreshold: fp('0.02').add(CBETH_ORACLE_ERROR).toString(), // 3%
```

Comment:
I think cbETH should use the combined oracle error, I'm seeing that the two oracle prices get multiplied together.

Reply:
It uses the two oracles for `low` and `high`, but only one for `pegPrice` which is what we use for `defaultThreshold`. Should we still consider both oracles in the default threshold calculation?

Reply:
Ah got it, my bad